### DISCUSSION TITLE
monitor data format error by prometheus

### DISCUSSION THREAD

**Discussion by Corey-Wang** [bfenetworks/bfe#1184] [General]
**Describe the bug**
/monitor/bal_table_status?format=prometheus
/monitor/host_table_status?format=prometheus
/monitor/proxy_mem_stat?format=prometheus

response data is not prometheus format

example:
/monitor/bal_table_status?format=prometheus
response
![image](https://user-images.githubusercontent.com/1393365/93996257-a01e9e80-fdc4-11ea-8ccc-e7ecdd93bb97.png)

**Comment by kwanhur** [comment]
From the [monitorHandlers](https://github.com/bfenetworks/bfe/blob/develop/bfe_websocket/server_tester.go#L47), we can known that `host_table_status` `host_table_version` `cluster_table_version` `bal_table_status` `bal_table_version` `module_status` `module_handlers` `proxy_mem_stat` non-support `prometheus` format, because they're not state related monitor items.
